CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - Stop sharing the same attributes hash between the fetched record and the memoized cache, which could interfere with dirty tracking
 
+#### 0.3.2
+
+- Fetch association returns relation or array depending on the configuration
```

Comment:
We might also want to mention that it was only returning a relation for cache_has_many fetch association methods